### PR TITLE
KEP-3221: Promote StructuredAuthorizationConfiguration to GA

### PR DESCRIPTION
**Pull request by liggitt**
### Description

Adds docs for GA of https://github.com/kubernetes/enhancements/issues/3221

/sig auth
/cc @ritazh @palnabarun

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates/structured-authorization-configuration.md [hunk 1]
```diff
@@ -6,13 +6,16 @@ _build:
...
-  - stage: alpha 
+  - stage: alpha
     defaultValue: false
     fromVersion: "1.29"
     toVersion: "1.29"
-  - stage: beta 
+  - stage: beta
     defaultValue: true
     fromVersion: "1.30"
+  - stage: stable
```
Comment (dipesh-rawat) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/structured-authorization-configuration.md:16]: `toVersion` should be added to the previous stage.

```suggestion
    toVersion: "1.31"
  - stage: stable
```

Comment (liggitt) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/structured-authorization-configuration.md:17]: done

**Pull request merged**